specs/features/step_definitions/authentication.js

```diff
@@ -86,7 +85,7 @@ defineSupportCode(({Given, Then, When}) => {
   const logout = () => {
     return client
       .isVisible('a.logout', (result) => {
-        client.click('a.logout').pause(1000)
+        client.click('a.logout')
```

Comment:
We should `.waitForElementVisible('.consent-acceptance', 5000)` here so that subsequent steps don't fail.

Reply:
I updated in the code but it's not reflecting here 🤷‍♂️